### ISSUE TITLE
[Bug] web_middleware list causes errors

### ISSUE THREAD

**Issue by hirakujira** [Laravel-Backpack/CRUD#3102]
In [https://github.com/Laravel-Backpack/CRUD/issues/3094](https://github.com/Laravel-Backpack/CRUD/issues/3094), @pxpm gave me some suggestions about this.

However when I check the code, I still feel confused.

In [https://github.com/Laravel-Backpack/CRUD/blob/master/src/config/backpack/base.php](https://github.com/Laravel-Backpack/CRUD/blob/master/src/config/backpack/base.php), there are two middleware groups, one is `admin`, and another one is `web`. 

As you can see, in [https://github.com/Laravel-Backpack/CRUD/blob/master/src/routes/backpack/custom.php](https://github.com/Laravel-Backpack/CRUD/blob/master/src/routes/backpack/custom.php
), the route loads both `web` and `admin` middleware groups.

So the problem is, @pxpm 's solution works for the `admin` middleware group only, if I want to make a custom array for `web` middleware in `/config/backpack/base.php`, it would cause errors. And also, modifying `web_middleware` to array object is what the code recommends me to do (please look the comments in the code)

https://github.com/Laravel-Backpack/CRUD/blob/d9e6201350260006ecbf4dec087a8c7639666eaf/src/config/backpack/base.php#L178-L222

_Originally posted by @hirakujira in https://github.com/Laravel-Backpack/CRUD/issues/3094#issuecomment-669015458_

**Comment by pxpm** [comment]
@hirakujira Thank you very much for submitting the bug report. 

Indeed at first I did not fully understand your question, my bad, sorry for that. 

I acknowledge the error your are facing and submited a PR: #3134 that I think would fix this.

**It would be super** if you could try out my solution on that PR and let me know there if it fixed your problem. 

Do not forget `php artisan config:cache` and `php artisan cache:clear` just to make sure you pick up the correct config. 

I am going to close this as we have a solution for it in #3134 and it's better to keep any discussion related with this in that thread. 

Once again, sorry for not understanding at first your issue and for the frustration of following the documentation and it didn't work. 

Hope we fix this for good :)

Wish you the best,
Pedro

**Comment by tabacitu** [comment]
Hi guys,

Reopening this, sorry 😄, because I haven't been able to reproduce the error. 

I've tried uncommenting the `web_middleware` array in the config. Adding stuff to it. Putting the array directly inside the route file. I cleared the cache of course. And **they worked just fine for me - even if there's an array inside an array, Laravel picks up the middleware, it seems to flatten the array**.

![2020-08-20 08 10 07](https://user-images.githubusercontent.com/1032474/90768619-fe83c900-e2bc-11ea-8f2c-e36f0ca1f85e.gif)

Are you guys sure there's a problem here? 
@pxpm were you able to reproduce @hirakujira 's issue and get the same error (`Illegal offset type in isset or empty`)?

Cheers!
